syndeo-template/pom.xml

```diff
@@ -355,6 +355,12 @@
             <version>1.17.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <version>4.2.0</version>
```

Comment:
very nit: our standard way has been <awaitility.version> on properties and just ${awaitility.version} here

Reply:
Done